span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/rawspan/ByPassPredicate.java

```diff
@@ -0,0 +1,31 @@
+package org.hypertrace.core.spannormalizer.rawspan;
+
+import com.typesafe.config.Config;
+import org.apache.kafka.streams.kstream.Predicate;
+import org.hypertrace.core.datamodel.AttributeValue;
+import org.hypertrace.core.datamodel.RawSpan;
+import org.hypertrace.core.spannormalizer.TraceIdentity;
+
+public class ByPassPredicate implements Predicate<TraceIdentity, RawSpan> {
+  private static final String SPAN_BYPASSED_CONFIG = "processor.bypass.key";
+  private String bypassKey;
+
+  public ByPassPredicate(Config jobConfig) {
+    bypassKey =
+        jobConfig.hasPath(SPAN_BYPASSED_CONFIG) ? jobConfig.getString(SPAN_BYPASSED_CONFIG) : null;
+  }
+
+  @Override
+  public boolean test(TraceIdentity traceIdentity, RawSpan rawSpan) {
+    AttributeValue defaultAttributeValue = AttributeValue.newBuilder().setValue("false").build();
```

Comment:
u can make this a constant or just return false

Reply:
part of the test so will do in follow up.